### PR TITLE
Remove verbose log prints to make the build/test log clean

### PR DESCRIPTION
**Pull request by NvTimLiu**
Fixes issue https://github.com/NVIDIA/spark-rapids/issues/2098

Remove the below tar verbose log prints in Jenkins to make the build/test log clean

tar -zxvf /home/ubuntu/spark-rapids-ci.tgz -C spark-rapids
CHANGELOG.md
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
NOTICE
NOTICE-binary
README.md
api_validation/
api_validation/auditAllVersions.sh
api_validation/src/
api_validation/src/main/
api_validation/src/main/resources/

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: jenkins/databricks/clusterutils.py [hunk 1]
```diff
@@ -42,10 +42,10 @@ def generate_create_templ(sshKey, cluster_name, runtime, idle_timeout,
         templ['autotermination_minutes'] = idle_timeout
         templ['enable_elastic_disk'] = 'false'
         templ['enable_local_disk_encryption'] = 'false'
-        templ['node_type_id'] = worker_node_type
+        templ['node_type_id'] = driver_node_type
         templ['driver_node_type_id'] = driver_node_type
         templ['ssh_public_keys'] = [ sshKey ]
-        templ['num_workers'] = num_workers
+        templ['num_workers'] = 0
```
Comment (jlowe) [inline jenkins/databricks/clusterutils.py:48]: These changes seem unrelated to the PR?

Comment (NvTimLiu) [reply jenkins/databricks/clusterutils.py]: Oops, let me droped it, seems I need to push -f to update the PR
Thanks @jlowe

**Comment by NvTimLiu** [comment]
build

File: jenkins/deploy.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ if [ "$DATABRICKS" == true ]; then
     rm -rf deploy
```
Comment (jlowe) [inline jenkins/deploy.sh:40]: 2021 copyrights

File: jenkins/databricks/deploy.sh [hunk 3]
```diff
@@ -19,7 +19,7 @@ set -e
 rm -rf deploy
```
Comment (jlowe) [inline jenkins/databricks/deploy.sh:19]: 2021 copyrights

File: jenkins/deploy.sh (hunk 2 above)
Comment (NvTimLiu) [reply jenkins/deploy.sh:40]: fixed

File: jenkins/databricks/deploy.sh (hunk 3 above)
Comment (NvTimLiu) [reply jenkins/databricks/deploy.sh:19]: fixed

**Comment by NvTimLiu** [comment]
build

**Review by jlowe — APPROVED** [review]
